src/Icon/Icon.js

```diff
@@ -8,12 +9,14 @@ import cx from 'classnames';
   icon: t.maybe(t.Str),
   className: t.maybe(t.Str),
   style: t.maybe(t.Obj),
+  paths: t.maybe(t.refinements(t.Number, x => x - Math.floor(x) === 0 && x > 0)),
   onClick: t.maybe(t.Func)
 })
 export default class Icon extends React.Component {
 
   static defaultProps = {
     className: '',
```

Comment:
as you're already managing this component, please remove `style` and `className` from `defaultProps`